.github/workflows/CodeQL-Analysis.yml

```diff
@@ -6,8 +6,6 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-  schedule:
```

Comment:
Removed Schedule wrt to @NicolasCARPi 's comment - https://github.com/3dmol/3Dmol.js/pull/659#discussion_r1146629965